docs/content/docs/tutorials/emr-on-eks-example.md

```diff
@@ -0,0 +1,368 @@
+```
+export EMR_NAMESPACE=emr-ns
+echo "creating namespace for $SERVICE"
+kubectl create ns $EMR_NAMESPACE
+
+echo "creating iamidentitymapping using eksctl"
+eksctl create iamidentitymapping \
+   --cluster $EKS_CLUSTER_NAME \
+   --namespace $EMR_NAMESPACE \
+   --service-name "emr-containers"
+```
+**Expected outcome**
+```
+2022-08-26 09:07:42 [ℹ]  created "emr-ns:Role.rbac.authorization.k8s.io/emr-containers"
+2022-08-26 09:07:42 [ℹ]  created "emr-ns:RoleBinding.rbac.authorization.k8s.io/emr-containers"
+2022-08-26 09:07:42 [ℹ]  adding identity "arn:aws:iam::012345678910:role/AWSServiceRoleForAmazonEMRContainers" to auth ConfigMap
+```
+## Install EMR on EKS controller
+Now we can go ahead and install EMR on EKS controller. First, let's export environment variables needed for setup
+```
+export SERVICE=emrcontainers
+export RELEASE_VERSION=`curl -sL https://api.github.com/repos/aws-controllers-k8s/$SERVICE-controller/releases/latest | grep '"tag_name":' | cut -d'"' -f4`
+export ACK_SYSTEM_NAMESPACE=ack-system
+```
+We cam use Helm for the installation
+```
+echo "installing ack-$SERVICE-controller"
+aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
+helm install --create-namespace -n $ACK_SYSTEM_NAMESPACE ack-$SERVICE-controller \
+  oci://public.ecr.aws/aws-controllers-k8s/$SERVICE-chart --version=$RELEASE_VERSION --set=aws.region=$AWS_REGION
+```
+**Expected outcome**
+```
+NAME: ack-emrcontainers-controller
+LAST DEPLOYED: Fri Aug 26 09:05:08 2022
+NAMESPACE: ack-system
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+emrcontainers-chart has been installed.
+This chart deploys "public.ecr.aws/aws-controllers-k8s/emrcontainers-controller:v0.0.6".
+
+Check its status by running:
+  kubectl --namespace ack-system get pods -l "app.kubernetes.io/instance=ack-emrcontainers-controller"
+
+You are now able to create Amazon EMR on EKS (EMRContainers) resources!
+```
+#### Configure IRSA for emr on eks controller
+Once the controller is deployed, you need to setup IAM permissions for the controller so that it can create and manage resources using EMR, S3 and other API's. We will use [IAM Roles for Service Account](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) to secure this IAM role so that only EMR on EKS controller service account can assume the permissions assigned.
+Please follow [how to configure IAM permissions](https://aws-controllers-k8s.github.io/community/docs/user-docs/irsa/) for IRSA setup. Make sure to change the value for `**SERVICE**` to `**emrcontainers**`
+
+After completing all the steps, please validate annotation for service account before proceeding.
+```
+# validate annotation
+kubectl get pods -n $ACK_K8S_NAMESPACE
+echo "sleeping for 5sec for pod to get ready"
+sleep 5
```

Comment:
You can use [`kubectl wait`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#wait) instead of `sleep 5`

Reply:
sleep or wait is not needed anymore because we are using ACK docs (https://aws-controllers-k8s.github.io/community/docs/user-docs/irsa/) for IRSA setup.